features/bookmarks/bookmark_filter.feature

```diff
@@ -17,3 +17,61 @@ Feature: Filter bookmarks
     Then I should see "2 Bookmarks by recengine"
       And I should see "english work"
       And I should see "german work"
+
+  # FIXME: currently failing
+  Scenario: Filtering series bookmarks by word count with restricted works
+    Given a bookmark by "awesome_posterrr" of a mixed visibility series with fandom "What a fandom"
```

Comment:
@irrationalpie7 : fix the tests in this file

Reply:
Can we help at all ?

Reply:
I think my comment is outdated (CI seems to be passing) so I'll remove this comment